src/app/gene-sets/gene-sets.component.ts

```diff
@@ -84,8 +84,7 @@ export class GeneSetsComponent extends ComponentValidator implements OnInit {
         ) || denovoGeneSetTypes[0];
 
         if (selectedStudyTypes) {
-          this.defaultSelectedDenovoGeneSetId.push(selectedStudyTypes.datasetId +
-            '-' + selectedStudyTypes.personSetCollectionId + '-denovo-geneset');
```

Comment:
What is the purpose of this change? Different denovo gene set ids must be made for each person set collection id.
When the person set collection id is omitted this happens when on dataset with two person collection ids:
![image](https://github.com/user-attachments/assets/4903533f-dd19-4837-aa62-7c23a3bb0288)
Loading the first time denovo gene sets on the dataset opens all person set collection ids, because it can't differentiate between them. Loading query also always opens person set collections - even empty of checked boxes.
Expected state of the UI:
![image](https://github.com/user-attachments/assets/bb2509eb-9a38-489a-bf33-3021501effe0)